CondFormats/BeamSpotObjects/interface/SimBeamSpotHLLHCObjects.h

```diff
@@ -0,0 +1,106 @@
+#ifndef SIMBEAMSPOTHLLHCOBJECTS_H
```

Comment:
Well, perhaps if you are going to touch the code anyhow, you could rename the header guard here with the usual inclusion of the system/package names

Reply:
ok, since I have to modify the PR in any case I'll change this!

Reply:
done in d4a9b65c7490af18d27ee8875090d46474455396